.travis.yml

```diff
@@ -18,6 +18,9 @@ gemfile:
   - gemfiles/Gemfile.rails-4.2.rb
   - gemfiles/Gemfile.rails-5.0.rb
 
+before_install:
+  - gem update bundler
```

Comment:
> It's confusing that this fails CI
 
@parndt , the problem was here. Travis used very old version of `bundler ` gem.

See https://travis-ci.org/norman/friendly_id/jobs/275578558#L982 :

```
$ bundle --version
Bundler version 1.8.4
```